CONTRIBUTING.md

```diff
@@ -1,19 +1,47 @@
-# Don't use the issue tracker to ask questions
```

Comment:
Happy to move the acceptance test parts back up to the top or to the wiki -- whatever you think is best.